lib/pdksync.rb

```diff
@@ -73,76 +70,76 @@ def self.main(steps: [:clone], args: nil)
         next unless exit_status.zero?
       end
       if steps.include?(:pdk_update)
         Dir.chdir(main_path) unless Dir.pwd == main_path
         next unless pdk_update(output_path).zero?
         if steps.include?(:use_pdk_ref)
-          ref = return_template_ref
+          ref = Utils.return_template_ref
           pr_title = module_args[:additional_title] ? "#{module_args[:additional_title]} - pdksync_#{ref}" : "pdksync_#{ref}"
           module_args = module_args.merge(branch_name: "pdksync_#{ref}",
                                           commit_message: pr_title,
                                           pr_title: pr_title,
-                                          pdksync_label: configuration.default_pdksync_label)
+                                          pdksync_label: Utils.configuration.default_pdksync_label)
         end
         PdkSync::Logger.info 'pdk update'
       end
       if steps.include?(:create_commit)
         Dir.chdir(main_path) unless Dir.pwd == main_path
         git_instance = Git.open(output_path)
-        create_commit(git_instance, module_args[:branch_name], module_args[:commit_message])
+        Utils.create_commit(git_instance, module_args[:branch_name], module_args[:commit_message])
         PdkSync::Logger.info 'commit created'
       end
       if steps.include?(:push)
         Dir.chdir(main_path) unless Dir.pwd == main_path
         git_instance = Git.open(output_path)
-        if git_instance.diff(git_instance.current_branch, "#{configuration.push_file_destination}/#{configuration.create_pr_against}").size != 0 # Git::Diff doesn't have empty? # rubocop:disable Style/ZeroLengthPredicate
+        if git_instance.diff(git_instance.current_branch, "#{Utils.configuration.push_file_destination}/#{Utils.configuration.create_pr_against}").size != 0 # Git::Diff doesn't have empty? # rubocop:disable Style/ZeroLengthPredicate
           PdkSync::Logger.info 'push'
-          push_staged_files(git_instance, git_instance.current_branch, repo_name)
+          Utils.push_staged_files(git_instance, git_instance.current_branch, repo_name)
         else
           PdkSync::Logger.info 'skipped push'
         end
       end
       if steps.include?(:create_pr)
         Dir.chdir(main_path) unless Dir.pwd == main_path
         git_instance = Git.open(output_path)
-        if git_instance.diff(git_instance.current_branch, "#{configuration.push_file_destination}/#{configuration.create_pr_against}").size != 0 # Git::Diff doesn't have empty? # rubocop:disable Style/ZeroLengthPredicate
-          pdk_version = return_pdk_version("#{output_path}/metadata.json")
+        if git_instance.diff(git_instance.current_branch, "#{Utils.configuration.push_file_destination}/#{Utils.configuration.create_pr_against}").size != 0 # Git::Diff doesn't have empty? # rubocop:disable Style/ZeroLengthPredicate
+          pdk_version = Util.return_pdk_version("#{output_path}/metadata.json")
```

Comment:
Hey @logicminds ,

NameError: uninitialized constant PdkSync::Util
It should be replaced with Utils

Cheers